credit_card.go

```diff
@@ -0,0 +1,61 @@
+package easypost
+
+import (
+	"context"
+	"errors"
+	"strings"
+)
+
+type CreditCard struct {
+	ID              string `json:"id,omitempty"`
+	DisbledAt       string `json:"disabled_at,omitempty"`
+	Object          string `json:"object,omitempty"`
+	Name            string `json:"name,omitempty"`
+	Last4           string `json:"last4,omitempty"`
+	ExpirationMonth string `json:"expiration_month,omitempty"`
+	ExpirationYear  string `json:"expiration_year,omitempty"`
+	Brand           string `json:"brand,omitempty"`
+}
+
+// FundCreditCard allows you to fund your EasyPost wallet by charging your primary or secondary card on file.
+func (c *Client) FundCreditCard(amount string, primaryOrSecondary string) (out *CreditCard, err error) {
+	return c.FundCreditCardWithContext(context.Background(), amount, primaryOrSecondary)
+}
+
+// FundCreditCardWithContext performs the same as FundCreditCard, but
+// allows specifying a context that can interrupt the request.
+func (c *Client) FundCreditCardWithContext(ctx context.Context, amount string, primaryOrSecondary string) (out *CreditCard, err error) {
+	paymentMethod, _ := c.ListPaymentMethods()
+	var cardId string
+
+	switch primaryOrSecondary {
+	case "primary":
+		cardId = paymentMethod.PrimaryPaymentMethod.ID
+	case "secondary":
+		cardId = paymentMethod.SecondaryPaymentMethod.ID
+	default:
+		return out, errors.New("payment method must be either primary or secondary")
```

Comment:
Please add a test that checks this branch. Should be simple since it makes no API calls and can be easily asserted against.

Reply:
yes, added